dvc/repo/experiments/queue/workspace.py

```diff
@@ -110,8 +113,12 @@ def _reproduce_entry(
             # Checkpoint errors have already been logged
             return {}
         except DvcException:
+            if self._EXEC_NAME == exec_name:
```

Comment:
We could do this inside the `finally`?

Reply:
At the beginning I put this in finally, but got something wrong with a `checkpoint` test. But looks like it was because I didn't remove the whole folder.